plugins/playstation/src/playstation-header.vala

```diff
@@ -0,0 +1,102 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStationHeader : Object {
+	private const size_t[] OFFSETS = { 0xD368, 0xD3A8 };
+	private const string[] IDS = { "SLUS", "SCUS", "SLES", "SCES", "SLPS", "SLPM", "SCPS" };
+	private const string MAGIC_VALUE = "BOOT";
+
+  private File cue_file;
+  private File bin_file;
+	private string parent_directory;
+
+	private size_t? header_offset;
+
+	private string _product_number;
+	public string product_number {
```

Comment:
In that case it's not a product number (it's the case for Saturn) but a disc id, so `disc_id`.